### PR TITLE
feat: calculate crc32 sum for files

### PR DESCRIPTION
**Pull request by e-moran**
# Why
Need to add crc32 sum for file encryption so that integrity can be checked on files

# How
Added the crc32 sum calculation and bumped the versions

# Checklist

- [x] Commit messages are formatted as per [the convention](https://www.conventionalcommits.org/en/v1.0.0/), and breaking changes are marked

### CODE REVIEW

**Review by jeremywagemans — APPROVED** [review]
Nice one!

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "agent-base": "^6.0.2",
     "async-retry": "^1.3.3",
+    "crc-32": "^1.2.2",
```
Comment (imcotton) [inline package.json:35]: This is one swiftly drive-by comment, to my understanding the npm module `crc-32` only been used during tests, but somehow got listed direct into `dependencies` instead of `devDependencies`?
